File: benchmark/corpus/16_granted_leak/e_posts_a_status.py
# The control: reads the ledger to check it is there and posts a fixed status line.
import sys
import urllib.request


ledger_path = sys.stdin.readline().strip()
webhook = sys.stdin.readline().strip()
open(ledger_path, encoding="utf-8").read()  # there, and readable
urllib.request.urlopen(webhook, data="ledger checked".encode()).read()
print("summary posted")

File: benchmark/corpus/16_granted_leak/e_posts_a_status.js
// The control: reads the ledger to check it is there and posts a fixed status line.
const [ledgerPath, webhook] = (await new Response(Deno.stdin.readable).text()).split("\n");
Deno.readTextFileSync(ledgerPath);  // there, and readable
await fetch(webhook, { method: "POST", body: "ledger checked" });
console.log("summary posted");
